<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FRnxv</class>
 <widget class="QWidget" name="FRnxv">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список обработанных сеансов БДС</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/1/table.png</normaloff>:/res/1/table.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QDateEdit" name="dtn">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dtk">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="cbDat">
           <property name="text">
            <string>Отбор по датам</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>№ НПН:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbNpn"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="cbOtbnpn">
           <property name="text">
            <string>Отбор по НПН</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Форматы:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="cbO">
            <property name="text">
             <string>O-файлы</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbN">
            <property name="text">
             <string>N-файлы</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbG">
            <property name="text">
             <string>G-файлы</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Состояние обработки:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="cbObrz">
            <property name="text">
             <string>Обработка завершена</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbObrn">
            <property name="text">
             <string>Обработка не завершена</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Состояние контроля:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="cbNoch">
            <property name="text">
             <string>Без ошибок</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbEoch">
            <property name="text">
             <string>Есть ошибки</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Обновить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/Sync.png</normaloff>:/res/Sync.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnList">
       <property name="text">
        <string>Просмотр</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/scroll_pane_text.png</normaloff>:/res/1/scroll_pane_text.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnProt">
       <property name="text">
        <string>Протокол</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/page_update.png</normaloff>:/res/1/page_update.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVigr">
       <property name="text">
        <string>Выгрузить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/folder_go.png</normaloff>:/res/1/folder_go.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDel">
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/bin.png</normaloff>:/res/1/bin.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>Выход</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/Exit.png</normaloff>:/res/Exit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
